src/markdown-pages/docs/managers/mq-manager.md

```diff
@@ -0,0 +1,233 @@
+---
+path: "/docs/managers/mq-manager"
+title: "MQ Manager"
+---
+
+**Alpha**
+
+## Overview
+This Manager provides the ability to connect a test to an existing IBM MQ queue manager, enabling one or more messages to be written to and read from existing queues. <br><br> 
+
+## Annotations
+
+The following annotations are available with the MQ Manager
+<details>
+<summary>MQ Manager</summary>
+
+| Annotation: |MQ Manager |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @QueueManager |
+| Description: | The <code>@QueueManager</code> annotation represents the name of the IBM MQ queue manager |
+| Attribute: `tag` |  Specifies which queue manager to connect to. Default value is _PRIMARY_. |
+| Syntax: | @QueueManager<br> public IMessageQueueManager qmgr;<br> |
+| Notes: | The <code>IMessageQueueManager</code> interface enables connection to the IBM MQ queue manager.  |
+</details>
+
+<details>
+<summary>MQ Queue</summary>
+
+| Annotation: |MQ Queue |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @Queue |
+| Description: | The <code>@Queue</code> annotation represents the name of the IBM MQ queue |
+| Attribute: `queueMgrTag` |  Specifies which queue manager to connect to. Default value is _PRIMARY_.  |
+| Attribute: `name` |  Specifies the name of the queue as it appears on the queue manager. Use the `name` attribute if the queue name never changes, regardless of environment.  |
+| Attribute: `tag` |  The name of the queue. Use the `tag` attribute when the value of the queue name is defined by a property in the CPS.  |
+| Notes: | You must specify either the `name` or the `tag` attribute but not both. If you specify both or neither, an exception is generated. |
+| Attribute: `archive` |  Archive messages retrieved and sent messages from and to this queue into the RAS to aid debugging. Default value is _true_. Valid values are _true_ and _false_. |
+| Syntax: | @Queue<br> public IMessageQueue queue;<br> |
+| Notes: | The <code>IMessageQueue</code> interface enables the test to put the provided messages onto the IBM MQ queues and retrieve messages from the IBM MQ queues.  |
+</details>
+
+
+
+## Code snippets
+
+Use the following code snippets to help you get started with the MQ Manager. Use the MQ Manager to connect a test to an existing IBM MQ queue manager and queue. Galasa does not provision new IBM MQ queue managers or queues. 
+ 
+<details><summary>Create an instance of an IBM MQ Manager</summary>
+
+The following snippet shows the code that is required to create an instance of an IBM MQ Manager:
+
+```java
+@QueueManager
+public IMessageQueueManager qmgr;
+```
+</details>
+
+<details><summary>Instantiate an IBM MQ queue</summary>
+
+The following snippet shows the code that is required to instantiate an IBM MQ queue:
+
+```java
+@QueueManager()
+public IMessageQueueManager qmgr;
+
+@Queue(archive = "true", name = "GALASA.INPUT.QUEUE")
```

Comment:
need to remove the quotes around the value true.  It was a String , but I've changed it to a boolean so the quotes are no longer needed

Reply:
removed